lib/pronto/eslint_npm.rb

```diff
@@ -60,13 +60,13 @@ def repo_path
     end
 
     def inspect(patch)
+      lines = patch.added_lines
       offences = run_eslint(patch)
       clean_up_eslint_output(offences)
         .map do |offence|
-          patch
-            .added_lines
-            .select { |line| line.new_lineno == offence['line'] }
```

Comment:
To go with less changes, shouldn't it be enough to change this line to ...

```rb
.select { |line| (offence['line']..(offence['endLine'] || offence['line']).cover?(line.new_lineno) }
```

... and this would be the only necessary change? Can't test it though, can you please test it?

Or is there any benefit from refactoring that method in addition?